x-pack/plugin/graph/qa/with-security/build.gradle

```diff
@@ -12,25 +13,15 @@ task copyGraphRestTests(type: Copy) {
   include 'rest-api-spec/test/graph/**'
 }
 
-integTestCluster {
-  dependsOn copyGraphRestTests
-  extraConfigFile 'roles.yml', 'roles.yml'
-  setupCommand 'setupTestAdminUser',
-               'bin/elasticsearch-users', 'useradd', 'test_admin', '-p', 'x-pack-test-password', '-r', 'superuser'
-  setupCommand 'setupGraphExplorerUser',
-               'bin/elasticsearch-users', 'useradd', 'graph_explorer', '-p', 'x-pack-test-password', '-r', 'graph_explorer'
-  setupCommand 'setupPowerlessUser',
-               'bin/elasticsearch-users', 'useradd', 'no_graph_explorer', '-p', 'x-pack-test-password', '-r', 'no_graph_explorer'
-  setting 'xpack.license.self_generated.type', 'trial'
+integTest.dependsOn copyGraphRestTests
+
+testClusters.integTest {
+  distribution = "DEFAULT"
```

Comment:
Do we strictly need to use the default distribution here? We should be judicious with this since it's much less expensive to use the integ-test distribution.

Reply:
No, but this is what was done with ClusterFormationTasks. 
I'm +1 on using integ-test and adding only the plugins and modules that are strictly needed for the test but  I would look at that separately and discuss it more broadly as all of x-pack is in the same boat.